dotCMS/src/main/java/com/dotcms/security/apps/AppSecretsImportExport.java

```diff
@@ -0,0 +1,21 @@
+package com.dotcms.security.apps;
+
+import java.io.Serializable;
+import java.util.List;
+import java.util.Map;
+
+public class AppSecretsImportExport implements Serializable {
+
+    static final long serialVersionUID = 1L;
+
+    private Map<String, List<AppSecrets>> secrets;
```

Comment:
Codacy found an issue: [Private field 'secrets' could be made final; it is only initialized in the declaration or constructor.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5931733)